docs/conf/portland/2022/news/welcome.rst

```diff
@@ -0,0 +1,93 @@
+:template: {{year}}/generic.html
+
+.. post:: Dec 15, 2022
+   :tags: {{shortcode}}-{{year}}, website, tickets
+
+Announcing Write the Docs {{ city }} {{year}}
+===========================================
+
+Welcome to the official launch of `Write the Docs Portland {{year}} <https://www.writethedocs.org/conf/portland/{{year}}/>`_.
+We're excited to announce the dates for our next conference:
+**Write the Docs {{ city }} {{ year }} will be on {{ date.short }}**
+
+We are planning to do a **hybrid event** this year,
+which means the in-person part of the conference will be returning.
+We are still in the planning phases for how to handle this safely,
+and would love your `feedback <mailto:portland@writethedocs.org>`_ on how we should progress.
+
+.. TODO: Add link to survey?
+
+It's been a rough year for a lot of us,
+but we're hoping that late next spring things will be looking a lot safer for us to gather.
+We plan to release more details during the next few weeks,
+but there's a few specific updates below.
+
+Health Guidelines
+-----------------
+
+We will continue to monitor the health guidelines for Portland & Oregon as the conference approaches,
+but our current plan is to have the following rules in place:
+
+* Vaccination required for all attendees
+* Masks required for all indoor spaces
+* A new venue that provides more access to outside air, and outdoor space
+* 100% refund for all attendees who need to cancel for COVID-related reasons
+* Limited venue capacity to give people enough room to socially distance
+* Seating options based on comfort level being near others in various spaces
+* Venue spaces with windows that open or that are fully outside
```

Comment:
Something something ventlation instead of talking about windows?

Reply:
Yup, it sounds like some windows are fully outside... ;) 

```suggestion
* Indoor meeting spaces with open windows (weather permitting)
* Outdoor meeting areas
```